tests/language-feature/modules/unscoped-enum-visibility.slang

```diff
@@ -0,0 +1,29 @@
+//TEST:COMPARE_COMPUTE(filecheck-buffer=CHECK):-cpu -output-using-type
```

Comment:
put these two tests into a sub directory, e.g. `tests/language-feature/modules/unscoped-enum-visibility/unscoped_enum_user.slang` and `tests/language-feature/modules/unscoped-enum-visibility/unscoped_enum_lib.slang`